### PR TITLE
 add weather v0.0.2 with autoupdate config

### PR DESCRIPTION
**Pull request by dannyxx001**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/noazark/weather
 * Official website (optional, not the repository): http://weatherjs.com/
 * NPM package url (optional): https://www.npmjs.com/package/weather.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 93
   - Count of watchers: 9
   - Count of forks: 47
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 26
   - Downloads in the last month: 94

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
You mixed two different libs here.

**Comment by dannyxx001** [comment]
@PeterDaveHello please help me check this pull request again, thank you.

File: ajax/libs/weather/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/noazark/weather.git"
+  },
+  "licenses": [
+    {
+      "type": "MIT"
+    }
+  ],
```
Comment (PeterDaveHello) [inline ajax/libs/weather/package.json]: Please use `"license": "MIT"` instead.

File: ajax/libs/weather/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "readme": "Weather.js\n==========",
+  "dist": {
+    "shasum": "201c0d53c19bedfaf42f21dbd3bc957caf57389d",
+    "tarball": "http://registry.npmjs.org/weather.js/-/weather.js-0.0.2.tgz"
+  }
```
Comment (PeterDaveHello) [inline ajax/libs/weather/package.json]: Please remove `"readme"` and `"dist"`

**Comment by dannyxx001** [comment]
@PeterDaveHello please help me check this pull request again, thank you.

File: ajax/libs/weather/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "basePath": "",
+      "files": [
+        "weather*"
+      ]
+    }
+  ],
+  "filename": "weather.min.js",
+  "version": "0.0.2",
+  "description": "Weather.js ==========",
```
Comment (PeterDaveHello) [inline ajax/libs/weather/package.json]: Please use something meaningful here, maybe the description on GitHub will be better (real weather for Javascript)`.

**Comment by PeterDaveHello** [comment]
Once you finished it, please rebase your branch for this PR with our latest master branch, thanks.

**Comment by dannyxx001** [comment]
@PeterDaveHello would you please help me check this pull request again, thank you.

**Comment by PeterDaveHello** [comment]
Should should cc its author, not issuer.

**Comment by dannyxx001** [comment]
OK, I have changed cc to its author.

**Comment by PeterDaveHello** [comment]
Thanks for your contribution.

**Pull request merged**